akka-typed/src/main/scala/akka/typed/javadsl/Actor.scala

```diff
@@ -201,6 +200,15 @@ object Actor {
     immutable(jSame, handler)
   }
 
+  /**
+   * Behavior that stops all child actors, waits for them to stop and then
+   * runs some user defined cleanup before stopping itself.
```

Comment:
I'd prefer "the user defined cleanup" over "some" which sounds like random cleanup it picks up somewhere

Reply:
sure